src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -59,7 +59,8 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
         });
 
         mapControl = new MapControl({
-            tabControl: tabControl
+            tabControl: tabControl,
+            isMobile: $(window).width() < MD_UP_BREAKPOINT
```

Comment:
The comparison needs to be done repeatedly in case the view resizes. For the other usage of this breakpoint at least, it's important to check for resize.